zeroize/src/aarch64.rs

```diff
@@ -8,13 +8,13 @@ use crate::{atomic_fence, volatile_write, Zeroize};
 use core::arch::aarch64::*;
 
 macro_rules! impl_zeroize_for_simd_register {
-    ($(($type:ty, $vdupq:ident)),+) => {
+    ($($type:ty),* $(,)?) => {
         $(
             #[cfg_attr(docsrs, doc(cfg(target_arch = "aarch64")))]
```

Comment:
Looks like this can be removed too?

Reply:
`uint8x8_t` and others are available only on AArch64 targets, so I don't think so.

Reply:
I guess this crate hasn’t been moved to `doc_auto_cfg` yet